process/indexer/lruOutportBlockCache_test.go

```diff
@@ -0,0 +1,191 @@
+package indexer
+
+import (
+	"encoding/hex"
+	"fmt"
+	"strings"
+	"sync"
+	"testing"
+
+	"github.com/multiversx/mx-chain-core-go/core/check"
+	"github.com/multiversx/mx-chain-core-go/data/outport"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewLRUOutportBlockCache(t *testing.T) {
+	t.Parallel()
+
+	t.Run("should work", func(t *testing.T) {
+		cache, err := NewLRUOutportBlockCache(1000)
+		require.Nil(t, err)
+		require.False(t, check.IfNil(cache))
+	})
+
+	t.Run("should work", func(t *testing.T) {
```

Comment:
"invalid cache size - should error"

Reply:
This part was removed since we added Extract and removed the lru part from cache